packages/playwright-core/src/server/page.ts

```diff
@@ -495,6 +496,16 @@ export class Page extends SdkObject {
 
   setScreencastOptions(options: { width: number, height: number, quality: number } | null) {
     this._delegate.setScreencastOptions(options).catch(e => debugLogger.log('error', e));
+    this._frameThrottler.setEnabled(!!options);
```

Comment:
Let's also cleanup (setEnabled(false)) when the page is closed, to avoid setInterval stuck forever.